.github/workflows/release.yml

```
@@ -34,7 +34,7 @@ jobs:
       - name: Zip firmware binaries (Nano)
         run: zip -r assets/hotdog-fw-nano-bin.zip hotdog-fw-nano-bin
       - name: Copy firmware sketch into assets
-        run: cp hotdog-fw/hotdog-fw.ino assets
+        run: cp Source/Firmware/Firmware.ino assets
```

This would also affect the release asset URL and break links to it. Please keep the old name.
```suggestion
        run: cp Source/Firmware/hotdog-fw.ino assets
```

---

Since you use the Arduino IDE and not something like PlatformIO where 
changing a folders name is no issue, I adjusted the firmwares filename.

Couldn't I just update the link with your next release?

---

Ah, true
Then could you please do [this](https://github.com/Sauceke/hotdog/pull/2#discussion_r912049697) instead